docs/customizing.md

```diff
@@ -245,3 +245,275 @@ Example `pulp_install_plugins` (with bogus version values):
       pulp-rpm:
          version: "7.8.9"
 ```
+
+Deployment Scenarios
+--------------------
```

Comment:
Another note about this entire section: I had an unstaged commit with further fixes to the example code and instructions. I remember getting pulled onto something else before I could issue a website PR for it.

We should follow up later about this, after you do the newsletter post, and after I finish my urgent SELinux issue.